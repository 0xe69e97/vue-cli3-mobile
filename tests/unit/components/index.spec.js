import { mount } from '@vue/test-utils'
import Test from '@/components/test'

describe('测试组件', function () {
  const asd = mount(Test)
  it('测试 test 组件的值', () => {
    expect(asd.vm.count).toBe(0)
    const button = asd.find('button')
    button.trigger('click')
    expect(asd.vm.count).toBe(28)
    expect(asd.element).toMatchSnapshot()
  })
})
