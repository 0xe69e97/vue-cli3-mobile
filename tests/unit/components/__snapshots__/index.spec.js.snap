// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试组件 测试 test 组件的值 1`] = `
<div>
  <button>
    +
  </button>
</div>
`;
